### PR TITLE
112632-task-card-styling

### PR DESCRIPTION
**Pull request by park9140**
112632
### Review
 - [ ] @boskya

### After Merge

run:

```
git checkout master
git pull
npm version patch
```

### note

If npm version is `2.13` or later npm version will likely fail to run postversion script, leaving you with pending commits in your repo.

If this is the case, run `npm run-script postversion` to finish the process.

### CODE REVIEW

File: styles/buttons/_mixins.scss [hunk 1]
```diff
@@ -0,0 +1,3 @@
+@mixin tap-highlight-color($color) {
+    -webkit-tap-highlight-color: $color;
```
Comment (boskya) [inline styles/buttons/_mixins.scss]: why do we need this non standards property?

File: styles/card/_styles.scss [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+    display: flex;
+    flex-direction: column;
+    flex-grow: 1;
+    flex-wrap: wrap;
+    margin: 0 1px 3px;
+    min-height: $card-height;
+    padding: $card-v-padding $card-padding;
+    text-align: left;
+
+    .item-title {
```
Comment (boskya) [inline styles/card/_styles.scss]: may be just title? or if more specificity is needed, then .card-title?

File: styles/card/_styles.scss [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+        line-height: $card-text-height-larger;
+        margin-bottom: 10px;
+        padding-left: 12px;
+        padding-right: 48px;
+        text-indent: -12px;
+
+        .item-number {
```
Comment (boskya) [inline styles/card/_styles.scss]: may be .card-number?

File: styles/card/_styles.scss [hunk 4]
```diff
@@ -0,0 +1,73 @@
...
+        .name {
+            @include font-size-large();
+        }
+    }
+
+    .related-title {
+        @include font-size-base-light();
+        font-style: italic;
+        line-height: $card-text-height-larger;
+        padding-right: 48px;
```
Comment (boskya) [inline styles/card/_styles.scss]: looks like you are using 48px multiple time...pull it out in a var

File: styles/card/_styles.scss [hunk 5]
```diff
@@ -0,0 +1,73 @@
...
+        margin-bottom: $flag-margin-bottom;
+        margin-right: $flag-margin-right;
+        margin-top: $flag-padding-top;
+        width: $flag-width;
+
+        &:hover {
+            cursor: pointer;
+        }
+
+        .off .image {
```
Comment (boskya) [inline styles/card/_styles.scss]: You probably didn't write this but a q - why do we need .image class here?

File: styles/card/_styles.scss [hunk 6]
```diff
@@ -0,0 +1,73 @@
...
+    .status {
+        @include font-size-base();
+        line-height: $card-text-height-large;
+    }
+
+    .end {
+        text-align: right;
+    }
+
+    .flag {
```
Comment (boskya) [inline styles/card/_styles.scss]: I am wondering if flag belongs here or in iconography? thoughts?

File: styles/card/_styles.scss (hunk 5 above)
Comment (joebri) [inline styles/card/_styles.scss]: Doesn't appear that we do! Removed.

File: styles/card/_styles.scss (hunk 6 above)
Comment (joebri) [inline styles/card/_styles.scss]: I would say that this flag is pretty tightly related to the task card. Although, we can review this again if future needs change.

File: styles/card/_styles.scss (hunk 5 above)
Comment (joebri) [inline styles/card/_styles.scss]: Apparently, this is required for some sort of IE issue. Am going to add comments which describe this).

**Pull request merged**